comid/rawvalue.go

```diff
@@ -0,0 +1,67 @@
+// Copyright 2021 Contributors to the Veraison project.
+// SPDX-License-Identifier: Apache-2.0
+
+package comid
+
+import (
+	"encoding/json"
+	"fmt"
+)
+
+type RawValue struct {
+	val interface{}
+}
+
+type TaggedRawValueBytes []byte
```

Comment:
Understand that the TaggedRaw Value is also Byte stream, but could you please also add what tag sits at the srtart?

Reply:
done in e1e9225